packages/addons/addon-depends/docker/cli/package.mk

```diff
@@ -24,7 +24,7 @@ configure_target() {
   export VERSION=${PKG_VERSION}
   export BUILDTIME="$(date --utc)"
 
-  cat > "${PKG_BUILD}/go.mod" << EOF
+  cat >"${PKG_BUILD}/go.mod"  <<EOF
```

Comment:
The script appears to have issues with the use of "<<", or I don't understand what style rule it's applying. Each "<<" it changes is adding a second space. This is just one example.

Reply:
good catch didn't saw that (for a while not even after you mentioned it), fixed (i push a commit EOF fix)